packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -263,49 +315,7 @@ class DateTimePicker extends React.Component {
 
   handleBlur(event, momentDateTime) {
     if (this.props.onBlur) {
-      const isCompleteDateTime = DateTimeUtils.isValidDateTime(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds);
-      let value = '';
-      if (this.dateValue) {
-        value = this.dateValue.concat(' ');
-      }
-
-      if (this.timeValue) {
-        value = value.concat(this.timeValue);
-      }
-
-      value = value.trim();
-
-      let tempDateTime = momentDateTime ? momentDateTime.clone() : null;
-
-      if (DateUtil.isValidDate(this.dateValue, this.state.dateFormat)) {
-        const enteredDateTime = DateTimeUtils.convertDateTimeStringToMomentObject(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds);
-
-        // this.state.dateTime does not get updated if the entered date is outside the minDate/maxDate range or an excluded date.
-        // In this case, we need to use the date that was entered instead of the this.state.dateTime.
-        if (enteredDateTime && !enteredDateTime.isSame(tempDateTime, 'day')) {
-          tempDateTime = enteredDateTime;
-        }
-      }
-
-      let iSOString = '';
-
-      if (isCompleteDateTime && tempDateTime) {
-        iSOString = tempDateTime.format();
-      }
-
-      let isValid = false;
-
-      if (value === '' || (isCompleteDateTime && tempDateTime && this.isDateTimeAcceptable(tempDateTime))) {
-        isValid = true;
-      }
-
-      const options = {
-        iSO: iSOString,
-        inputValue: value,
-        isCompleteValue: isCompleteDateTime,
-        isValidValue: isValid,
-      };
-
+      const options = this.getMetadata(momentDateTime);
```

Comment:
Let's rename `options` to `metaData` 🙃

Reply:
Addressed here : https://github.com/cerner/terra-framework/pull/1045/commits/a1449c5a19dbeb60ff1189d49f9838ecc5d38e3d